db/migrate/20160524234655_create_student_identifiers.rb

```diff
@@ -0,0 +1,20 @@
+class CreateStudentIdentifiers < ActiveRecord::Migration
+  def change
+
+    create_table :student_identifier_types do |t|
+      t.belongs_to :organization, index: true
+      t.string :name
+      t.string :description
+      t.integer :content_type
+    end
+
+    create_table :student_identifiers do |t|
+      t.belongs_to :organization, index: true
+      t.belongs_to :user,         index: true
+      t.belongs_to :student_identifier_type
+      t.string :value
+    end
```

Comment:
Should these column values ever be `NULL` if that's the case we should enforce this on the database as well as in the Rails model.